pixie/stdlib.pxi

```diff
@@ -166,8 +166,11 @@
                           (recur)))))))))
 
 
-(def reduce (fn [rf init col]
-              (-reduce col rf init)))
+(def reduce (fn 
+              ([rf col]
+               (reduce rf (first col) col))
```

Comment:
I think this should be `(reduce rf (first col) (next col))`, Otherwise this happens: `(= (reduce + [1 2]) 4)`